sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -78,7 +78,7 @@ Q.submit([&](handler& cgh) {
 ...
 ```
 
-In this example, the SYCL compiler needs to compile the lambda expression passed
+In this example, the compiler needs to compile the lambda expression passed
```

Comment:
Is there some real value to remove SYCL everywhere from here?

Reply:
In a few places like that (when it can be applied well enough to any other SYCL/DPC++ compiler or ) I just tried to be more generic by saying just compiler.